frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -117,6 +188,15 @@ export function DynamicConfiguration () {
         }
     };
 
+    const addApplication = async (detail: any) => {
```

Comment:
Why is this async?

Reply:
Not sure, now that I think of it we'd want this to be synchronous